com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/build.gradle

```diff
@@ -53,6 +53,7 @@ dependencies {
     api fileTree(dir: 'libs', include: ['*.jar'])
     implementation chartboostMediationDependency
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4")
+    implementation("com.google.code.gson:gson:2.10.1")
```

Comment:
kotlin has a deserialization library that we can use. Perhaps, we can give that a try?

Reply:
🤔  Yeah, I can look into it

Reply:
Ended up using `JSONObject`